app/views/jobseekers/job_applications/build/declarations.html.slim

```diff
@@ -21,3 +21,6 @@
 
       = f.govuk_submit t("buttons.continue") do
         = f.govuk_submit t("buttons.save_as_draft"), secondary: true
+
+  .govuk-grid-column-one-third
+    = render(Shared::ProcessStepsComponent.new(process: job_application.vacancy, service: process_steps, title: t("jobseekers.job_applications.process_title")))
```

Comment:
I assume that `title: t("jobseekers.job_applications.process_title")` can similarly be chopped down to `title: t("process_title")` if you move process_title to be under jobseekers.job_applications.build in the jobseekers.yml. 

However, I can't test this, because I can't see any process steps on the page when I check out the branch.

I put a breakpoint in `ProcessStepsComponent.render?` and discovered that `@process` was a vacancy - the one I was applying to - rather than the application. So `@process.state` was `"edit_published"`.

Reply:
t("process_title") doesnt work in this case - you would have to have a process_title defined for each step, so leaving as is